src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookMultibranchPRTrigger.java

```diff
@@ -0,0 +1,61 @@
+package com.atlassian.bitbucket.jenkins.internal.trigger;
+
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
+import com.google.common.annotations.VisibleForTesting;
+import hudson.Extension;
+import hudson.model.Item;
+import hudson.triggers.Trigger;
+import hudson.triggers.TriggerDescriptor;
+import jenkins.branch.MultiBranchProject;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+public class BitbucketWebhookMultibranchPRTrigger extends Trigger<MultiBranchProject<?, ?>> {
+
+    private static final Logger LOGGER = Logger.getLogger(BitbucketWebhookMultibranchPRTrigger.class.getName());
+
+    @SuppressWarnings("RedundantNoArgConstructor") // Required for Stapler
+    @DataBoundConstructor
+    public BitbucketWebhookMultibranchPRTrigger() {
+    }
+
+    @Override
+    public BitbucketWebhookMultibranchPRTrigger.DescriptorImpl getDescriptor() {
+        return (BitbucketWebhookMultibranchPRTrigger.DescriptorImpl) super.getDescriptor();
+    }
+
+    @Symbol("BitbucketWebhookMultibranchPRTriggerImpl")
```

Comment:
I'm uncertain we need this? Given the symbol should be "short name that can be easily typed in by humans" and this one is longer than the class name, I wonder if we should reconsider either the use of `@Symbol` or shorten it.